components/CategorySelection/index.js

```diff
@@ -23,15 +23,18 @@ const CategoriesSelection = ({ url, title }) => {
       </div>
       <div className="container mx-auto">
         <div className="flex flex-wrap items-center justify-center font-bold">
-          {data.data.map((item) => (
-            <Card
-              key={item.id}
-              id={item.id}
-              title={item.name}
-              url={item.imgUrl}
-              link={`${item.name}`}
-            />
-          ))}
+          {categories.data.map((category) => {
+            let { id, name, imgUrl } = category;
+            return (
+              <Card
+                key={id}
+                id={id}
+                title={name}
+                url={imgUrl}
+                link={`${name}`}
+              />
+            );
+          })}
```

Comment:
It's possible to do this destructuring on line 26 as well if you wanted to do with `map(({ id, name, imgUrl }) => ...`

Is there any particular reason why link needs to use a template string? It might make sense if you need to put a `/` before the name to generate a link for example, but just wanted to check.

Reply:
Hi @Alun,  I had no idea! Thank you. It looks way better now.
I changed the name to slug, Is this better now?

Reply:
Hmm... you're going to have to tell me the answer to that. I guess that the `link` prop is going to mean that when the user clicks on card, they get redirected to that url? If that's the case, you need to make sure that the link says the right thing otherwise the link won't work. I don't know what your endpoints are, or what this data looks like, so you're better placed to answer this than me